### ISSUE TITLE
Question papers

### ISSUE THREAD

**Issue by GetPsyched** [isac-nitkkr/node-modules#36]
Well well well, we're here finally implementing [what NKSS dreamt of](https://drive.google.com/drive/folders/12pGDoyfxrBXkv7kPoPIoXuU2ec2cEL0z).

Okay so there's a lot to uncover here; as always, only authorised personnel can upload question papers. However! Students will be able to make submissions that can be then reviewed by a team and merged into our knowledge base.

Having that out of the way, here's a WIP TODO:
- [ ] A schema that encapsulates a QP entity (part of #15)
- [ ] Authorised question paper CRUD (depends on #28)
- [ ] Once submitted, it backlogs for screening which has its own page
- [ ] Attribution to whoever submitted the paper

Maybe in the future we can convince the faculty to generate papers from the website instead of word documents; eliminating the need to add papers here in the first place. For now, we don't have the power to do this.
